### DISCUSSION TITLE
在线获取SPOTIFY_REFRESHTOKEN

### DISCUSSION THREAD

**Discussion by aaro-n** [DIYgod/RSSHub#9388] [Show and tell]
简单记录获取spotify refresh token
1.在spotify创建应用，获取Spotify 应用的 client ID和Spotify 应用的 client secret
2.打开这个[网址](https://getyourspotifyrefreshtoken.herokuapp.com)，找到spotify创建的应用，将重新定向URL设置** https://getyourspotifyrefreshtoken.herokuapp.com/callback  **。
3.将Spotify 应用的 client ID和Spotify 应用的 client secret填入对应的位置，权限全部勾选，点击“Submit”。
4. 此时spotify会弹出确认对话框，点击确认就可在下方获取refresh token。

**Comment by Freddy23317** [comment]
there is nothing here,yet。
网站失效了吗

**Reply by TonyRL** [reply]
Use https://alecchendev.github.io/spotify-refresh-token/ from https://github.com/alecchendev/spotify-refresh-token or https://spotify-refresh-token-generator.netlify.app/ from https://github.com/limhenry/spotify-refresh-token-generator.
